R/statistic-histogram.R

```diff
@@ -53,14 +75,12 @@ histogram.getAccuracy <- function(mechanism, n.bins, n, epsilon, delta=10^-6, al
 #'    Default to 10^-6.
 #' @param alpha A numeric vector of length one specifying the numeric 
 #'    statistical significance level. Default to 0.05.
-#' @param error The error term of the statistical significance level. Default
-#'    to 1e-9. 
 #' 
-#' @export histogram.getParameters
+#' @export histogram.getEpsilon
 #' @return Differential privacy parameter epsilon
-#' @rdname histogram.getParameters
+#' @rdname histogram.getEpsilon
 
```

Comment:
Maybe add a note here saying this calculation is the inverse of the analysis for histogram.getAccuracy.